app/assets/javascripts/discourse/app/models/user.js

```diff
@@ -779,18 +779,26 @@ const User = RestModel.extend({
     }
   },
 
-  updateNotificationLevel(level, expiringAt) {
+  updateNotificationLevel(level, expiringAt, sourceUser = null) {
```

Comment:
Do we use "sourceUser" elsewhere in the app? I think we use "actingUser", but I might be wrong.

Reply:
> There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.

Thanks, I will change to `actingUser` here and in the controller, it's much better :)